internal/config/validate.go

```diff
@@ -72,7 +72,17 @@ func (v ValidationResults) Error() string {
 	return fmt.Sprintf("%d validation errors: %s", len(v), strings.Join(values, ", "))
 }
 
-const AppeningHappening = "[DEPRECATION] More than one app stanza within a waypoint.hcl file is deprecated, and will be removed in 0.10.\nPlease see https://discuss.hashicorp.com/t/deprecating-projects-or-how-i-learned-to-love-apps/40888 for more information."
+func (v ValidationResults) Errors() bool {
```

Comment:
`Errors()`->`HasErrors()`? 

A method `Errors()` makes me think I'm going to get a slice of errors returned, if any exist. I think `HasErrors()` matches the intent here better, similar to the HCL `HasErrors()` in other parts of this file

Reply:
Sure, good call. Changing now.